lab.js

```diff
@@ -0,0 +1,50 @@
+document.addEventListener("DOMContentLoaded", () => {
+    fetch("https://cors-anywhere.herokuapp.com/https://official-joke-api.appspot.com/random_ten").then(res => {
+        if(!res.ok){
+            throw Error(res.statusText)
+        }
+        return res.json()
+    }).then(jokes => {
+        let ul = document.createElement("ul")
+        jokes.forEach(joke => {
+            let li = document.createElement("li")
+            li.innerText = joke.setup;
+            // let punchline = document.createElement("li")
+            // punchline.innerText = joke.punchline
+            ul.appendChild(li)
+        })
+        document.body.appendChild(ul)
+    }).catch(err => console.log(err))
+})
+
+// const loadJokes = () => {
+//     fetch("https://cors-anywhere.herokuapp.com/https://official-joke-api.appspot.com/random_ten").then(res => {
+//         if(!res.ok){
+//             throw Error(res.statusText)
+//         }
+//         return res.json()
+//     }).then(jokes => {
+//         let ul = document.createElement("ul")
+//         jokes.forEach(joke => {
+//             let li = document.createElement("li")
+//             li.innerText = joke.setup
+//             ul.appendChild(li)
+//         })
+//         document.body.appendChild(ul)
+//     }).catch(err => console.log(err))
+// }
+
+const createCardFromJoke = (joke) => {
```

Comment:
This looks good, you can add event listeners to the setup divs that execute this function. Although adding event listeners to each li wouldnt be the most effective way to handle this.